packages/frontend/src/context/StoreContext.tsx

```diff
@@ -92,7 +96,30 @@ export const StoreContextProvider = (
         stateManager.eventStreamProcessing.catch((e) => {
           debug(`Error while executing eventStreamProcessing ${e}`);
         });
+        stateManager.seqNo.on("seqNo", (res) => {
```

Comment:
why are we storing the last seen seqNo in localStorage an not in level/index with the events?